src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1527,8 +1527,8 @@
   <Node Name="BoundConvertedTupleLiteral" Base="BoundTupleExpression">
     <!-- Converted tuple must have a type -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
-    <!-- Natural type is preserved for the purpose of semantic model. Can be null -->
-    <Field Name="NaturalTypeOpt" Type="TypeSymbol" Null="allow"/>
+    <!-- Original tuple is preserved for the purpose of semantic model -->
+    <Field Name="SourceTuple" Type="BoundTupleLiteral" SkipInVisitor="true" ForceInNullabilityRewriter="true" />
```

Comment:
>ForceInNullabilityRewriter [](start = 76, length = 26)

Perhaps `UpdateFieldInNullabilityRewriter` instead, so it doesn't sound like it conflicts with `SkipInNullabilityRewriter`

Reply:
Or perhaps we can update `SkipInVisitor` to accept more than just true/false

---
In reply to: [296056473](https://github.com/dotnet/roslyn/pull/36628#discussion_r296056473) [](ancestors = 296056473)